tests/graceful-recovery/graceful-recovery.md

```diff
@@ -56,9 +56,7 @@ to deploy NGINX Gateway Fabric using manifests and expose it through a LoadBalan
 
 9. In a different terminal, deploy the
 [https-termination example](https://github.com/nginxinc/nginx-gateway-fabric/tree/main/examples/https-termination).
-10. Inside the NGINX container, navigate to `/etc/nginx/conf.d` and check `http.conf` and `config-version.config` to see
```

Comment:
I removed all of the conf related checks because I don't think they are necessary. We check that the config is accurate by passing traffic.